sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ManagementGroup/Models/CreateManagementGroupRequest.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> Management group creation parameters. </summary>
+    public partial class CreateManagementGroupRequest
```

Comment:
General comment: seems like most of these generated classes should be internal.  Is there a reason why we are leaving them as public?

If they need to be public we should turn off EditorBrowsable, I think

Reply:
that one is public because it is passed into the PUT command, this resources doesn't use the normal construct that the [Resource]Data is the input / output object, they have separate RQ / RS objects